config/routes.rb

```diff
@@ -5,5 +5,6 @@
 
   resources :organizations, only: %i[show] do
     resources :users, only: %i[index], module: :organizations
+    resources :subjects, only: %i[index new create edit update destroy], module: :organizations
```

Comment:
Can we simplify this line?
```suggestion
    resources :subjects, module: :organizations
```

Reply:
We can write `except: %i[show]`, but i don't really like `except` option, i think thats ok

Reply:
Oops, I forgot about `show`